style/swift/sample.swift

```diff
@@ -85,3 +85,13 @@ struct DopeObject {
     return DopeObject(json: json)
   }
 }
+
+// Break up long function calls and initializers
+let foo = Foo(
+  bar: Bar(),
+  baz: Baz(),
+  quz: Quz(
+    x: 42,
+    y: 365
+  )
```

Comment:
Always struggled with knowing what to do with the closing bracket here. I sort of lean towards keeping it in the same line as the last argument:

```
let foo = Foo(
  bar: bar,
  baz: baz)
```